### ISSUE TITLE
Getting the altitude/elevation from a single point

### ISSUE THREAD

**Issue by supertobi** [bopen/elevation#19]
I would like to get the elevation for a single point with the latitude and longitude.

I use darktable[1] to geotag photos. Darktable ha a nice map function to do this with drag and drop [2], bud sadly this function doesn't insert the elevation to the metadate. Now I would like to write a script that adds the elevation to all photos with latitude and longitude.

[1] https://www.darktable.org/
[2] https://www.darktable.org/usermanual/ch05.html.php

**Comment by alexamici** [comment]
@supertobi please take a look at Elevation design document: https://github.com/bopen/elevation/blob/master/docs/design.rst

Getting the elevation of a single point from the command line is out of the current, very limited, scope of the project. Maybe at some point the scope will grow enough to revisit this, but this at the moment I don't see it happening any time soon.

I'll leave the issue open anyhow.

**Comment by supertobi** [comment]
@alexamici that's OK. Do you know any other tool to get the elevation from a single point but doesn't need to register to a special service like google?

**Comment by alexamici** [comment]
@supertobi not really, and I also don't think you can reliably ask for the elevation of a "point", you always need to define the size of the point, so that you can assess whether the resolution of your elevation data is enough.
